### PR TITLE
worker disk space cleanup

### PR DESCRIPTION
**Pull request by oren-zohar**
### Summary of your changes
Fixing K8s CI issue by cleaning up disk space in the runner

### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

**Comment by oren-zohar** [comment]
Seems like it adds around 4 minutes to the CI, I'll see how can I optimize it first

**Comment by orestisfl** [comment]
> Seems like it adds around 4 minutes to the CI, I'll see how can I optimize it first

The `large-packages` option of the action you were using before might help speed up if disabled: https://github.com/jlumbroso/free-disk-space/blob/4d9e71b726748f254fe64fa44d273194bd18ec91/action.yml#L171-L186

**Comment by oren-zohar** [comment]
> > Seems like it adds around 4 minutes to the CI, I'll see how can I optimize it first
> 
> The `large-packages` option of the action you were using before might help speed up if disabled: https://github.com/jlumbroso/free-disk-space/blob/4d9e71b726748f254fe64fa44d273194bd18ec91/action.yml#L171-L186

Saved 5 seconds :metal:

File: .github/workflows/eks-ci.yml [hunk 1]
```diff
@@ -65,7 +65,6 @@ jobs:
           android: true
           dotnet: true
           haskell: true
-          large-packages: true
```
Comment (orestisfl) [inline .github/workflows/eks-ci.yml]: It defaults to `true`, just deleting it won't do anything

Comment (oren-zohar) [reply .github/workflows/eks-ci.yml]: Much faster. Let's see if the freed up space is enough
image

**Review by orestisfl — APPROVED** [review]

**Pull request merged**